src/test/kotlin/no/nav/syfo/mocks/GraphApiMock.kt

```diff
@@ -0,0 +1,41 @@
+package no.nav.syfo.mocks
+
+import com.auth0.jwt.JWT
+import io.ktor.client.engine.mock.*
+import io.ktor.client.request.*
+import io.ktor.http.*
+import no.nav.syfo.client.graphapi.GraphApiGroup
+import no.nav.syfo.client.graphapi.GraphApiUserGroupsResponse
+import no.nav.syfo.testhelper.UserConstants
+import no.nav.syfo.util.JWT_CLAIM_NAVIDENT
+
+private val adGroupRoleCoffee =
```

Comment:
Denne skal bare simulere en AD-gruppe som ikke har tilgang?

Reply:
mmnei, den er fyllmasse. Den er en del av tilgangene veilederen har, men ikke én tilgang vi bryr oss om.

Reply:
Men vi skriver om på variabelnavnene litt for å gjøre det tydeligere